tests/components/onewire/const.py

```diff
@@ -398,13 +396,22 @@
             },
             {
                 ATTR_DEFAULT_DISABLED: True,
-                ATTR_DEVICE_CLASS: DEVICE_CLASS_CURRENT,
-                ATTR_ENTITY_ID: "sensor.26_111111111111_current",
-                ATTR_INJECT_READS: b"       1",
-                ATTR_STATE: "1.0",
+                ATTR_DEVICE_CLASS: DEVICE_CLASS_VOLTAGE,
+                ATTR_ENTITY_ID: "sensor.26_111111111111_current_vis",
```

Comment:
Since the name was changed in the definition, it also needs to be adjusted in the test constants:
```suggestion
                ATTR_ENTITY_ID: "sensor.26_111111111111_vis",
```